### PR TITLE
AP-2024 error now works when police_notified is blank

### PR DESCRIPTION
**Pull request by eloisetait3**
## What

[Link to story](https://dsdmoj.atlassian.net/browse/AP-2024)

Form validation added to handle true or false selection. Translations also added.

When nothing selected OR text box is blank, the error is now correctly displayed at the top & at the relevant question level, ie the question is highlighted in red with an appropriate error message.

## Checklist

Before you ask people to review this PR:

- [x] Tests and rubocop should be passing: `bundle exec rake`
- [x] Github should not be reporting conflicts; you should have recently run `git rebase master`.
- [x] There should be no unnecessary whitespace changes. These make diffs harder to read and conflicts more likely.
- [x] The PR description should say what you changed and why, with a link to the JIRA story.
- [x] You should have looked at the diff against master and ensured that nothing unexpected is included in your changes.
- [x] You should have checked that the commit messages say why the change was made.

### CODE REVIEW

File: app/forms/respondents/respondent_form.rb [hunk 1]
```diff
@@ -51,6 +47,16 @@ def initialize(*args)
 
     private
 
+    def police_notified_presence
+      return if police_notified.blank?
+
+      value = __send__("police_notified_details_#{police_notified}".to_sym)
```
Comment (Alex-Swann) [inline app/forms/respondents/respondent_form.rb:53]: I think this still works without the need for `.to_sym`

File: app/forms/respondents/respondent_form.rb [hunk 2]
```diff
@@ -51,6 +47,16 @@ def initialize(*args)
...
+      return if value.present? || draft?
+
+      translation_path = "activemodel.errors.models.respondent.attributes.police_notified_details_#{police_notified}.blank"
+      errors.add("police_notified_details_#{police_notified}".to_sym, I18n.t(translation_path))
```
Comment (Alex-Swann) [inline app/forms/respondents/respondent_form.rb:57]: remove `.to_sym`

**Review by Alex-Swann — CHANGES_REQUESTED** [review]
Just one small comment otherwise looks good

File: app/forms/respondents/respondent_form.rb (hunk 1 above)
Comment (eloisetait3) [reply app/forms/respondents/respondent_form.rb]: done!

File: app/forms/respondents/respondent_form.rb (hunk 2 above)
Comment (eloisetait3) [reply app/forms/respondents/respondent_form.rb:57]: done

**Review by Alex-Swann — APPROVED** [review]

**Pull request merged**